python/grass/temporal/abstract_space_time_dataset.py

```diff
@@ -2872,7 +2853,7 @@ def update_from_registered_maps(self, dbif=None):
             dbif.execute_transaction(sql, mapset=self.base.mapset)
 
         # Count the temporal map types
-        maps = self.get_registered_maps_as_objects(dbif=dbif)
+        maps = self.get_registered_maps(order="start_time", dbif=dbif)
```

Comment:
Is `get_registered_maps_as_objects` still used anywhere else, or used publicly, or is part of our public API? If that was the last reference, there would be some cleaning to do.

Reply:
The function is relatively frequently used:
https://github.com/search?q=repo%3AOSGeo%2Fgrass%20get_registered_maps_as_objects(&type=code

https://github.com/OSGeo/grass/pull/3350/commits/125c80e6b197d643c955835f596d212edf399347 should improve the performance quite a bit...